.gitlab/common/macos.yml

```diff
@@ -28,8 +28,15 @@
       echo "Warning: The current Python version $PYTHON_VERSION is different from $PYTHON_REPO_VERSION in .python-version."
       echo "Installing Python $PYTHON_REPO_VERSION..."
     fi
+    ## START TEMPORARY DEBUG see ACIX-500
+    echo $PYTHON_VERSION
+    echo $VENV_NAME
+    pyenv virtualenvs --bare
```

Comment:
Can you list the content of `..../.pyenv/versions/` to see what is already there?

Reply:
Good point, fixed with [88ce984](https://github.com/DataDog/datadog-agent/pull/30759/commits/88ce9849045631a527d16eb9deb3ffe177b6481b)!